pkg/utils/miniojob/types.go

```diff
@@ -151,6 +151,11 @@ func (jobCommand *MinIOIntervalJobCommand) createJob(_ context.Context, _ client
 		}
 	}
 
+	if jobCR.Spec.Insecure != nil && *jobCR.Spec.Insecure && insecure == false {
```

Comment:
Condition says `insecure == false` but below we are setting
```
jobCommands = append(jobCommands, "--insecure")
insecure = true
```
Is this correct?

Reply:
I guess the `insecure` flag is set when the user explicitly has set the command to be insecure. However we should use `!insecure` instead of `insecure == false`:
```suggestion
	if jobCR.Spec.Insecure != nil && *jobCR.Spec.Insecure && !insecure {
```